### PR TITLE
Zarina article post

### PR DESCRIPTION
**Pull request by zmustapha**

### CODE REVIEW

File: content/articles/rebuilding-compiled.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+As always, the development and design of CompilED is an iterative process. There is a bit more to be done, for example, the search functionality has to be optimized for accessibility.
+
+More will be written soon about what I've learned in making sites for access and accessibility.
+
+## Structure, search, and share
+
+Hugo, similar to Movable Type v4.x,  doesn't come with a seach functionality. We could have set up a Google custom site search like we did in the old CompilED, but, according to Anders, "that would've limited our ability to customize the display and integration of search results and there would always be a delay between changes to content being made and Google re-indexing the site."
```
Comment (sdreher) [inline content/articles/rebuilding-compiled.md]: seach > search

File: content/articles/rebuilding-compiled.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+│   │   ├── article-post-files
+│   ├── projects
+│   │   ├── article-post-files
+│   ├── etcetera...
+│   │   ├── article-post-files
+```
+
+This structure made it difficult for me to use the existing Hugo list of content templates, layout template assigments, and other functionality such as global pagination. It also complicated the editorial process where contributors would have to drop their posts in a topic directory and move the files around if the topic changed.
+
+To simplify content and template management, I eliminated the topic directories, and turned the topics into taxanomies that are defined in each article's front matter. Here, an article's topic can be edited, deleted, or created at anytime without moving the file between directories. All posts are now in one directory, `articles`. This is the new content structure:
```
Comment (sdreher) [inline content/articles/rebuilding-compiled.md]: taxanomies > taxonomies

File: content/articles/rebuilding-compiled.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+title: "Rebuilding CompilED"
+topics:
+- Process
+type: post
+---
+
+*"We can rebuild [it]. We have the technology."*  
+— Oscar Goldman, *The Six Million Dollar Man*
+
+Earlier in March we decided to move our previous blog to a different platform, with a fresh design in hope to breathe new life into it, and keep it going painlessly.
```
Comment (sdreher) [inline content/articles/rebuilding-compiled.md]: simply imho. I'm finding this first sentence a bit awkward and long. Perhaps break into two sentences?

Comment (zmustapha) [inline content/articles/rebuilding-compiled.md]: It's fine. I'm not going to fix that sentence.

**Pull request merged**